tests/virt_autotest/virt_utils.pm

```diff
@@ -169,7 +169,7 @@ sub clean_up_red_disks {
 
     if (($disks_nu_length eq $disks_nu_num) && $disks_nu_num && $fs_type_supported) {
         foreach my $item (@disks_nu_array) {
-            if ($item =~ /\/dev\/sd[b-z].*/) {
+            if ($item =~ /^\/dev\/sd[b-z]{1,}$/) {
```

Comment:
+1 to @asdil12, I guess we either don't need it at all, or if disk name will have 4 letters, I doubt that they will have 'sd' prefix. So I guess it will be strict enough.

Reply:
I will update according to your comments. Get rid of ","

Reply:
Then you can remove repetition section completely, as [b-z] will match one character anyway.

Reply:
Will dot that also.